tools/cli/adminKafkaCommands.go

```diff
@@ -508,9 +502,6 @@ func doRereplicate(
 		maxID = maxRereplicateEventID
 	}
 
-	histV2 := cassandra.NewHistoryV2PersistenceFromSession(session, loggerimpl.NewNopLogger())
```

Comment:
i would simple suggest remove the admin kafka resend command all together.
this pile of code is unmaintainable and legacy

Reply:
Filed Issue number #701